api/app/.env.example

```diff
@@ -16,3 +16,7 @@ POSTGRES_PASSWORD="wps"
 POSTGRES_DATABASE="wps"
 POSTGRES_PORT="5432"
 PORT="8080"
+PATHFINDER_BASE_URI=https://console.pathfinder.gov.bc.ca:8443
+PROJECT_NAMESPACE=openshift_project_namespace
```

Comment:
No need to add an env variable for this because we already keep track of this (check out `openshift/scripts/oc_deploy.sh` line 29)